src/OrchardCore.Modules/OrchardCore.PublishLater/Views/_ViewImports.cshtml

```diff
@@ -0,0 +1,8 @@
+@inherits OrchardCore.DisplayManagement.Razor.RazorPage<TModel>
+
+@using OrchardCore.PublishLater.ViewModels
+@using OrchardCore.DisplayManagement
+
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+@addTagHelper *, OrchardCore.DisplayManagement
+@addTagHelper *, OrchardCore.ResourceManagement
```

Comment:
newline

Reply:
Can you elaborate?

Reply:
Every time you see a little circle icon in the end of your files that means it requires you to add a new line at the end for Git.

Reply:
This icon : 
![image](https://user-images.githubusercontent.com/3228637/75927142-c89b7700-5e39-11ea-96c2-3ef984c9c887.png)

Reply:
I think it's by default added whenever you are using a VS

Reply:
Ah OK, that's interesting, thank for clarifying. Reading about it seems to be a leftover from the 70s :D. I fixed these.